hic_qc.py

```diff
@@ -300,6 +308,25 @@ def extract_header_info(self, header, xx=50):
             for min_size in self.mapping_dict.keys():
                 self.contigs_greater[min_size] = set([contig['SN'] for contig in header['SQ'] if contig['LN'] > min_size])
 
+
+        if 'PG' in header and 'bwa' in header['PG'][0]['CL']:
```

Comment:
How do you decide to use 0 or 1 when accessing header['PG'][#]?  I assume it works right, but might be good to add a simple comment about the structure of the BAM headers you are parsing.

Reply:
I'm not fully sure of the logic underlying this, so just adding a #TODO